### DISCUSSION TITLE
Suppress suggestions for Get-Command

### DISCUSSION THREAD

**Discussion by powercode** [PowerShell/PowerShell#14999] [Q&A] [answered]
Is there a way to suppress command suggestions from `Get-Command` except using `-ErrorAction:Ignore`?
I would like to be able to get the error without the suggestions, but not have a suggestion in the output.

When run from a script, the suggestion can't be that useful, so it would be nice to at least be able to suppress them.

**Comment by vexx32** [comment, marked as answer]
I had a PR a while back that refactored suggestions into a mere property of the ErrorRecord itself; that would allow for that to work. Custom error view, or simple configuration, could be used to turn them off.

I never got around to finishing it, though. @SteveL-MSFT expressed interest in picking it up, but I don't think he's had time to get it sorted / other things took priority.

Until something like that happens, it's not really possible to simply disable them; the suggestions are sort of built into the shell itself, and the only way to remove them is to use a custom host afaik.
